cves/kernel/CVE-2016-0823.yml

```
@@ -169,8 +174,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: RFC violation.
```

A quick word about why an RFC violation is out of scope for auto-discovery would be good
```suggestion
  note: Vulnerability results from an RFC violation, which requires more domain knowledge to be identified than an autodiscovery tool would reasonably have.
```